python/tvm/script/scope_handler.py

```diff
@@ -185,92 +226,365 @@ def let(var, value, span):
         super().__init__(let, concise_scope=False, def_symbol=False)
 
 
+@register
+class Block(WithScopeHandler):
+    """ With scope handler tir.block(extents, name) as iter_vars"""
```

Comment:
I think we could really use more documentation on the usage and parameters of `tir.block`. I'm not sure where it should go though.

Reply:
Most documents are on the C++ side